packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -491,7 +540,7 @@ describe("calcite-list", () => {
     it("navigating items after filtering", async () => {
       const page = await newE2EPage();
       await page.setContent(html`
-        <calcite-list filter-enabled>
+        <calcite-list drag-enabled filter-enabled>
```

Comment:
Does this behave differently without `drag-enabled`? If so, we should add another test.

Reply:
It should have had drag-enabled there before. It shouldn't be setting those props if dragging isn't enabled.